lib/charms/mutual_tls_interface/v0/mutual_tls.py

```diff
@@ -0,0 +1,346 @@
+# Copyright 2023 Canonical Ltd.
```

Comment:
Please fetch the `certificate-exchange-interface` instead.

Reply:
Done.